src/index.js

```diff
@@ -95,6 +95,13 @@ module.exports = (
           if (!err.missing || !err.missing.length)
             return callback(err);
           // make not found errors runtime errors
```

Comment:
I think this comment "make not found errors runtime errors" is in the wrong place

Reply:
And it might be worth adding a comment in the request.endsWith('.js') to explain why this is necessary.

Reply:
This feature still applies separately - since if the path isn't found, we always ensure it is a runtime check.

This has been branched now as for backwards compat I wanted to first check the webpack resolver against `./file.js` before trying `./file` so it's effectively doing two resolutions checks on the fallback.

That said, I haven't guarded this by ensuring the parent is definitely a tsx? module.... let me add that.

Reply:
Fixed the comments.